app/views/employee_profiles/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>My profile</h1>
+
+  <p>First name: <%= current_employee.first_name %></p>
```

Comment:
This should be using @employee instead of current_employee as you are setting @employee in the controller. Same for the rest of the file